corehq/apps/groups/models.py

```diff
@@ -148,11 +159,7 @@ def get_static_users(self, is_active=True):
 
     @classmethod
     def by_domain(cls, domain):
```

Comment:
i have conflicted feelings about whether i think we should also make it clearer from these method names that the data might be stale. can see arguments both ways.